bitset.go

```diff
@@ -209,6 +209,27 @@ func (b *BitSet) Flip(i uint) *BitSet {
 	return b
 }
 
+// FlipRange bit in [start, end).
+// If end>= Cap(), this function will panic.
+// Warning: using a very large value for 'i'
```

Comment:
This comment seems out of place (there is no 'i' anywhere in the new code).

Reply:
I have fixed the comment 'i'